### DISCUSSION TITLE
After initial import of Takeout, can immich-go just add albums from a new takeout?

### DISCUSSION THREAD

**Discussion by ghanjiboy** [simulot/immich-go#535] [Q&A]
Hi, I have immich running and I previously imported a takeout export with immich-go successfully.  Since then I am still using both google photos and Immich.    So now the photos are being added to both at the moment.

If I get a new takeout export, I dont really need to import the actual photos, I just need the new albums created since the photos already exist in Immich - is this possible?

**Comment by simulot** [comment]
Yes, no problem.
Any duplicated photo will be ignored
